samples/excel/89-preview-apis/chart-trendlines.yaml

```diff
@@ -0,0 +1,203 @@
+
+                if (Office.context.requirements.isSetSupported("ExcelApi", 1.7)) {
+                    sheet.getUsedRange().format.autofitColumns();
+                    sheet.getUsedRange().format.autofitRows();
+                }
+
+                createChart(context);
+                sheet.activate();
+
+                await context.sync();
+            });
+        }
+
+        async function createChart(context: Excel.RequestContext) {
+            const sheet = context.workbook.worksheets.getItem("Sample");
+            const salesTable = sheet.tables.getItem("SalesTable");
+
+            const dataRange = salesTable.getDataBodyRange();
+
+            let chart = sheet.charts.add("ColumnClustered", dataRange, Excel.ChartSeriesBy.columns);
+
+            chart.setPosition("A15", "F30");
+            chart.title.text = "Quarterly sales chart";
+            chart.legend.position = "right"
+            chart.legend.format.fill.setSolidColor("white");
+            chart.dataLabels.format.font.size = 15;
+            chart.dataLabels.format.font.color = "black";
+            let points = chart.series.getItemAt(0).points;
+            points.getItemAt(0).format.fill.setSolidColor("pink");
+            points.getItemAt(1).format.fill.setSolidColor("indigo");
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            }
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+
+
+
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to add, set, get and format chart trendlines.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Add sample data</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="add-trendline-polynomial" class="ms-Button">
+                <span class="ms-Button-label">Add trendline</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <button id="get-trendline" class="ms-Button">
+                <span class="ms-Button-label">Get trendline type</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <button id="get-trendline-color" class="ms-Button">
+                <span class="ms-Button-label">Get trendline color</span>
+            </button>
+        </section>
+        <section class="samples ms-font-m">
+            <button id="set-trendline-color" class="ms-Button">
+                <span class="ms-Button-label">Format and set trendline color</span>
```

Comment:
Change button text to:  "Set trendline color"

Reply:
ok